pkg/policy/execute.go

```diff
@@ -252,9 +252,9 @@ func (e *Executor) createCheckResult(ctx context.Context, policyExecution *state
 	checkResult := &state.CheckResult{
 		ExecutionId:        policyExecution.Id,
 		ExecutionTimestamp: policyExecution.Timestamp,
-		Name:               qr.Name,
+		Name:               q.Name,
 		Selector:           normalizeCheckSelector(policyExecution, e.PolicyPath, q.Name),
-		Description:        qr.Description,
+		Description:        q.Title,
```

Comment:
In the policy execution we already map these fields to query result

https://github.com/cloudquery/cloudquery/blob/80f2b31dbd698d02ae0aa00ac4e54c5685b55d00/pkg/policy/execute.go#L335-L342